src/compiler/rust/canister/src/lib.rs

```diff
@@ -80,8 +85,6 @@ thread_local! {
     static RELOADED_JS_TIMESTAMP: RefCell<u64> = RefCell::new(0);
 
     static RELOADED_JS: RefCell<BTreeMap<u64, Vec<u8>>> = RefCell::new(BTreeMap::new());
-
-    static FILE_INFO: RefCell<BTreeMap<String, (Timestamp, BytesReceived, Hash, BytesHashed)>> = RefCell::new(BTreeMap::new());
 }
```

Comment:
Was this deleted or moved? If moved I think we should discuss keeping all global variables together in the same place, you can export them and import them to where they are used

Reply:
It was moved, it seemed nice to me to isolate all of the mutation of a global variable to one file. I'm not opposed to keeping it in one place though